.github/workflows/socrata_upload.yaml

```diff
@@ -62,13 +66,13 @@ jobs:
         env:
           AWS_IAM_ROLE_TO_ASSUME_ARN: ${{ secrets.AWS_IAM_ROLE_TO_ASSUME_ARN }}
           AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
-          AWS_ATHENA_S3_STAGING_DIR: ${{ secrets.AWS_ATHENA_S3_STAGING_DIR }}
-          AWS_REGION: ${{ secrets.AWS_REGION }}
+          AWS_ATHENA_S3_STAGING_DIR: ${{ vars.AWS_ATHENA_S3_STAGING_DIR }}
+          AWS_REGION: ${{ vars.AWS_REGION }}
```

Comment:
**[Question, non-blocking]** Elsewhere we just hardcode this to `"us-east-1"`, any reason not to do that here as well?

https://github.com/ccao-data/data-architecture/blob/ac90562f411757e53b74349f2eace76621f81f7d/dbt/scripts/run_iasworld_data_tests.py#L1206-L1210